packages/react/src/components/footer/footerBase/FooterBase.test.tsx

```diff
@@ -6,7 +6,9 @@ import { FooterWrapper } from '../../../utils/test-utils';
 
 describe('<Footer.Base /> spec', () => {
   it('renders the component', () => {
```

Comment:
Could we add a mock for the Date object? Without one we need to update the snapshot on every new year, since the copyright year is dynamic